firmware/Core/Src/unit_tests/test_telecommand_arg_helpers.c

```diff
@@ -1,6 +1,55 @@
 #include "telecommands/telecommand_args_helpers.h"
 #include "unit_tests/unit_test_helpers.h"
 
+uint8_t TEST_EXEC__TCMD_ascii_to_uint64() {
+    uint64_t result;
+
+    // Success: Nominal test
+    TEST_ASSERT(TCMD_ascii_to_uint64("123", 3, &result) == 0);
+    TEST_ASSERT(result == 123);
+
+    // Error: Empty string
+    TEST_ASSERT(TCMD_ascii_to_uint64("", 0, &result) == 1);
+
+    // Error: String is too long
+    TEST_ASSERT(TCMD_ascii_to_uint64("12345678912345678912", 20, &result) == 2);
+
+    // Success: String has a digit length of 19
+    TEST_ASSERT(TCMD_ascii_to_uint64("1234567891234567891", 19, &result) == 0);
+    TEST_ASSERT(result == 1234567891234567891);
+
+    // Error: String doesn't start with an integer
+    TEST_ASSERT(TCMD_ascii_to_uint64("abc123", 6, &result) == 3);
+
+    // Error: Result length doesn't match str_len
+    TEST_ASSERT(TCMD_ascii_to_uint64("123abc", 6, &result) == 3);
+
+    return 0;
+}
+
+uint8_t TEST_EXEC__TCMD_extract_uint64_arg() {
+    uint64_t result;
+    char str[] = "1,2,3,4,a,6,7,8,9";
+    // Success: Nominal Test
+    TEST_ASSERT(TCMD_extract_uint64_arg(str, sizeof(str), 0, &result) == 0);
+    TEST_ASSERT(result == 1);
+
+    char str2[0];
```

Comment:
I know that making an array of size 0 is illegal, however is this alright for the purpose of unit tests?

Reply:
I'd strongly advise against it. An array of size 1, with the argument currently passed at `sizeof(str2)` set explicity to 0, would be safer.

Best as I can tell, `char str2[0];` is undefined behaviour by the C standard, and relies on the given C compiler to implement well.

Reply:
Fixed!